strings_lab.js

```diff
@@ -0,0 +1,178 @@
+console.log(firstChar[0])
+//
+// 2. `console.log` the length of a string.
+console.log(firstChar.length)
+
+// 3. `console.log` the last character of any string.
+
+console.log(firstChar[firstChar.length-1])
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+let myName = "voniel"
+let lastLetter = myName[myName.length-1]
+//console.log(lastLetter)
+  upperLast = lastLetter.toUpperCase()
+let split = myName.slice(0,5)
+    console.log(split + upperLast)
+
+// 5. Create a drEvil code block that will take a single number variable, and log the '<variableAmount> dollars',
+// your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+let variableAmount = 1000000;
+
+if (variableAmount === 1000000){
+  console.log( variableAmount + " dollars " + "(pinky)")
+} else {
+  console.log(variableAmount + " dollars")
+}
+
+// 6. Create a `verbing` code block. It should take a single string variable.
+//If its length is at least 3, it should add 'ing' to its end, unless it already ends in 'ing', in which case it should add 'ly' instead. If the string length is less than 3, it should leave it unchanged.
+// For example:
+//
+// ```js
+// verbing('box')
+// // => 'boxing'
+// verbing('train')
+// // => 'training'
+// verbing('swimming')
+// // =>  'swimmingly'
+// verbing('go')
+// // =>  'go'
+// ```
+let verbing ="jump";
+if (verbing.endsWith("ing")){
+        let changeWord = verbing.replace("ing","ly")
+        console.log(changeWord)
+} else if (verbing.length >= 3){
+          console.log(verbing + "ing")
+}
+
+
+
+// 7. Create a `withoutLast` code block that takes a single string variable, and returns a copy of the string without the last word.
+
+let string = "itsy bitsy spider";
+
+let newString = string.slice(0,10)
+console.log(newString)
+
+// ## Bonus Questions
+//
+// 1. Write a code block that adds a "#" to a string if the string has an odd number of characters, otherwise leave it as is.
+// print/log the result
+//
+//     Examples:
+//
+//   ```js
+//   let name = 'taq'  // should log 'taq#'
+//   let name2 ='john' // should log 'john'
+
+let strName = "olive";
+
+if (strName.length % 2) {
+  console.log(strName + "#")
+} else {
+  console.log(strName)
+}
+
+
+
+// 2. Write a code block that prints/logs the 3rd to last characters of a string.
+// If the string is not at least 3 characters long then log an error message.
+// <details>
+//     <summary>
+//         Hint
+//     </summary>
+//     It should work for strings of any length.
+// </details>
+//
+// Examples, for variables:
+// ```js
+// let name1 = 'Taq' // Should log 'Taq'
+// let sentence = 'He is Taq'// Should log 'Taq'
+let sentence= "what do you say to the god of death ";
```

Comment:
If sentence is "what do you say to the god of death" then is should print out "ath". I think you understood question wrong. On the other hand, you created a code that searches which is useful in other ways. You will use "includes" a lot in the future so get familiar with that method.